setup.py

```diff
@@ -41,17 +41,17 @@ def open_local(paths, mode="r", encoding="utf8"):
         "Development Status :: 3 - Alpha",
         "Environment :: Web Environment",
         "License :: OSI Approved :: MIT License",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
     ],
 }
 requirements = [
-    "httpx==0.18.*",
-    "websockets>=9.0",
+    "httpx",
+    "websockets",
```

Comment:
I just tested it with httpx 0.20, but I am **SUPER** hesitant to unpin this version. Httpx has a long history of breaking changes. So, if we do I think we need to be prepared for the day when it _will_ break. Moreover, this lib will certainly not work with anything less than 0.18. As for `websockets`, we can probably remove that requirement completely since it will install with Sanic.

Reply:
Yeah, I ran into the same thing going from 0.19 to 0.20, particularly regarding 3xx status codes and the `follow_redirects` Client arguments. Completely understanding pinning while HTTPX is in 0.x versioning, so long as it doesn't block newer Sanic versions.

Reply:
Do you think it's reasonable to do 0.18-0.20? The range we know it works in? After the first of the year I have an idea for a better implementation where we shouldn't need to worry so much and only touch the high level API.

Reply:
Sure

Reply:
Since it doesn't seem like the 0.20 change impacted the tests here, I don't see why not.